manifests/longhorn.yaml

```diff
@@ -3784,4 +4289,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-      serviceAccountName: longhorn-service-account
\ No newline at end of file
+      serviceAccountName: longhorn-service-account
+---
+# Source: longhorn/templates/validate-psp-install.yaml
+#
```

Comment:
The PSP is missing. We should also think about replacing it against a PSA, maybe with a little help from a friend? https://github.com/kubernetes-sigs/pspmigrator